code/TemplateStudioForWinUICs/Templates/Ft/ThemeSelection/Services/ThemeSelectorService.cs

```diff
@@ -46,20 +46,19 @@ public async Task SetRequestedThemeAsync()
 
         private async Task<ElementTheme> LoadThemeFromSettingsAsync()
         {
-            ElementTheme cacheTheme = ElementTheme.Default;
-            string themeName = await _localSettingsService.ReadSettingAsync<string>(SettingsKey);
+            var themeName = await _localSettingsService.ReadSettingAsync<string>(SettingsKey);
 
-            if (!string.IsNullOrEmpty(themeName))
+            if (Enum.TryParse(themeName, out ElementTheme cacheTheme))
```

Comment:
Does this definitely still work if `themeName` is null (or empty)?

Reply:
Yes.